src/main/java/com/fdmgroup/backend_streamhub/authenticate/controller/AccountController.java

```diff
@@ -46,4 +57,26 @@ public ApiResponseAccount AccountUpdateTest(@RequestBody Account inputAccount) {
 
         return response;
     }
+
+    @PostMapping("/account/login/submit")
+    public ResponseEntity<String> registrationAttempt(@RequestBody LoginRequest loginRequest) {
+        try {
+            accountControllerLogger.info("Login attempt | {}.", loginRequest.toString());
+            accountService.loginUser(loginRequest);
+            return ResponseEntity.status(HttpStatus.OK).body("Login successful.");
```

Comment:
For successful login, can you return a JSON of the user info (userID, username, email) - this would be necessary for front end for tracking the logged-in user status

Reply:
I've made the changes. When you successfully register an account and test logging in on Postman, it returns a JSON.

Reply:
As we discussed on Teams, login is based on username and password only. Successful login attempts will return a JSON comprising of id and username.